### ISSUE TITLE
Data Set Data issues: Date fields

### ISSUE THREAD

**Issue by MichelleNevius** [abtassociates/LSASampleCode#154]
As I try to import the data set dated 2020_4_20, I encounter several places where the date values are not in the standard format. For example:
Export.csv, all in row 2:
ExportDate '4/30/2020 13:42' is not a valid value for ExportDate. ExportDate must contain a date and time in the format yyyy-mm-dd hh:mm:ss. 
ExportStartDate '10/1/2012' is not a valid value for ExportStartDate. ExportStartDate must contain a date in the format yyyy-mm-dd.
ExportEndDate '9/30/2019' is not a valid value for ExportEndDate. ExportEndDate must contain a date in the format yyyy-mm-dd.

Similarly, in ProjectCoC.csv, DateCreated and DateUpdated seems incorrectly formatted in each row.

Similarly, in Inventory.csv, the 4 data elements involving dates have the formatting issue: 
InventoryStartDate, InventoryEndDate, DateCreated, DateUpdated.

Would it be possible to get a new data set with these dates formatted correctly? Thanks!

**Comment by dburgstrom** [comment]
When the file is opened in a text editor, the data are formatted correctly:
"2018-02-14 14:31:07","2013-10-01","2017-09-30"
There is a footnote to the data types in the HMIS CSV specifications that speaks to this formatting issue when opening .csv files in Excel:
1. In CSV files opened with MS Excel, date and datetime data are automatically reformatted.  Even if a user makes no other changes, saving the file will save the dates in the Excel format.  It is tedious but possible to reset the dates to the CSV format using “Format Cells/Custom” menu options so that the file will pass validation checks.
Thanks.

**Comment by MichelleNevius** [comment]
Hi. Thanks for your reply. .... I wasn't opening the files in Excel. I was taking the zipped files and attempting an import into our application with no editing done to them (which works in our application with real data sets with no issues). So, I guess I have to try opening them in a text editor just to rezip them? That is unexpected but I can try.

**Comment by dburgstrom** [comment]
Is it possible for you to open one of those "real data sets" in a text editor side-by-side with the .csv file and compare the date formats? There could be some slight difference in formatting--something to cause the issue you see. Is the application you refer to your HMIS or a different application?

**Comment by bayboos** [comment]
Hello, I'm Miek Korwel from Foothold Technology - a programmer working with Michelle. I downloaded the current fileset and opened the Export.csv using Sublime Text - a text editor famous for (among other things) not interfering with the file content in any way. The second line of this file looks like this (I removed the beginning of this line for better readability):

> (...),jdoe@email.com,4/30/2020 13:42,10/1/2012,9/30/2019,HMIS,2020,3,2,4

As an example, here's the line from the Export.csv file generated by our software (truncated the same way as the line above):

> (...),"",2020-04-24 03:13:51,2016-10-01,2020-04-24,"AWARDS","2.6",3,2,1

As you can see, the difference between "test kit data" (the first line) and our "real life data" (the second line) is not slight by any means. In the test kit example, the dates are in the "US format" (mm/dd/yyyy) while the HMIS standards require ISO formatting (yyyy-mm-dd). Similar thing is true with the time format. Additionally, the non-numeric values are not in the double quotes as they should to 1) prevent the text editors (expecially the Microsoft-branded ones) from interferring with the values and 2) follow the HMIS CSV standards.

Thank you for taking the time to read this note; any response will be highly appreciated.

**Comment by dburgstrom** [comment]
Hi Miek, thank you for checking that and for your explanation. I looked again and see the issue on my end too. @MichelleNevius my apologies for mixing up my response, I must have been looking at the wrong file.

My process involved altering and removing some local characteristic data from these files, including Exit.csv; however I opened the files in Excel and saved them again. I will take my own advice and make any needed edits in Notepad++ (or Sublime Text) this time around. That is my fault. Thank you for bringing this to my attention.
